packages/gestalt/src/Masonry.js

```diff
@@ -93,6 +94,10 @@ type Props<T> = {|
    * Specifies whether or not Masonry dynamically adds/removes content from the grid based on the user's viewport and scroll position. Note that `scrollContainer` must be specified when virtualization is used.
    */
   virtualize?: boolean,
+  /**
+   * Experimental prop to batch paints for possible performance improvements. This is an experimental prop and may be removed in the future.
+   */
+  _batchPaints?: boolean,
```

Comment:
Props prefixed with an underscore are automatically filtered out of the props tables in the docs, so this will remain undocumented